test/website/index.html

```diff
@@ -62,6 +62,11 @@
         const apps = await raygunClient.Applications.get("", "");
         console.log("Applications", apps);
```

Comment:
This should be renamed to `Application` as that method will only retrieve a single app.